Blankmon/src/application/MoveResult.java

```diff
@@ -0,0 +1,27 @@
+package application;
+
+public class MoveResult
+{
+	private double mDamageDone;
+	private String mDialogueTxt;
+	
+	public MoveResult(double damageDone, String dialogueTxt)
+	{
+		if(dialogueTxt == null)
+			throw new IllegalArgumentException("dialogueTxt was null");
+		
+		mDamageDone = damageDone;
+		mDialogueTxt = dialogueTxt;
+	}
+
+	public double getDamageDone()
+	{
+		return mDamageDone;
+	}
+
+	public String getDialogueTxt()
+	{
+		return mDialogueTxt;
+	}
+	
+}
```

Comment:
I see you made the MoveResult class. Is there anything about this class you wish to change or do you like the implementation?

Reply:
I think its fine for now, though we'll probably add more to it and different variations as we add in items and switching.